src/custom/state/swap/trade.test.ts

```diff
@@ -0,0 +1,112 @@
+import { parseUnits } from '@ethersproject/units'
+import { getCanonicalMarket } from '@src/custom/utils/misc'
+import { PriceQuoteParams } from '@src/custom/utils/operator'
+import { basisPointsToPercent } from '@src/utils'
+import { ChainId, Fraction, JSBI, Pair, Route, Token, TokenAmount, Trade, TradeType, WETH } from '@uniswap/sdk'
+
+async function mockGetPriceQuote(params: PriceQuoteParams & { mockedPriceOut: string }) {
```

Comment:
why do you need this to be a function and have a somewhat complex definition if you then hardcode it with a fixed price and fixed token. I think it's easier to read if you have the quote already as a constant.